CMake/FollyCompilerUnix.cmake

```diff
@@ -26,6 +24,20 @@ function(apply_folly_compile_options_to_target THETARGET)
       -Wunused-label
       -Wunused-result
       -Wnon-virtual-dtor
-      ${FOLLY_CXX_FLAGS}
-  )
+      ${FOLLY_CXX_FLAGS})
+
+  if (APPLE)
+    target_compile_options(${THETARGET}
+      PRIVATE
+        -Wno-nullability-completeness # See https://github.com/facebook/folly/issues/740
+        -Wno-inconsistent-missing-override # See https://github.com/facebook/folly/pull/745
```

Comment:
I would vote for adding this flag the same way I did with -Wno-nullability-completeness in d7dfcff981b8.
We should use this flag whenever we are compiling with clang, not just on Mac platforms.

Reply:
Very valid point -- I should make it for Clang and not just Apple builds. Can easily add the compiler flag using `CHECK_CXX_COMPILER_FLAG` and will do so shortly. Thanks!